Libplanet.Tools/Mpt.cs

```diff
@@ -35,7 +43,7 @@ public class Mpt
         public void Diff(
             [Argument(
                 Name = "KEY-VALUE-STORE",
```

Comment:
How about shortening these terms too?

```suggestion
                Name = "KV-STORE",
```

Reply:
I'm not sure about it. Is this an abbreviation used so often?

Reply:
[I believe so.](https://www.google.com/search?q=kv+store)

Reply:
It is true.

Reply:
So would you shorten the term or not?

Reply:
I agree to shorten the term. And I applied it in 297dd30.

Reply:
`Argument` attributes' `Name` fields still use the longer term…

Reply:
It seems a mistake. I amended it in 81c10121310525525b3c3ae35ef40b803812be99 again.. Do you feel to amend other arguments' `Name` fields (e.g., `STATE-ROOT-HASH`) or enough?